projects/ion/src/lib/avatar/avatar.component.spec.ts

```diff
@@ -59,10 +59,26 @@ describe('Avatar', () => {
   });
   describe('Photo', () => {
     it('should render avatar with image when type is photo', async () => {
-      await sut({ type: AvatarType.photo, image: 'assets/images/avatar.jpg' });
+      await sut({
+        type: AvatarType.photo,
+        image:
+          'https://64.media.tumblr.com/40e2174ab5e68b1eabbc3dfc78607cef/c1effc67d5c3a1fd-20/s540x810/9d6ce72fcddf97841e7410a0652dd9d5f018b35d.pnj',
+      });
+      expect(screen.getByRole('img')).toHaveAttribute(
+        'src',
+        'https://64.media.tumblr.com/40e2174ab5e68b1eabbc3dfc78607cef/c1effc67d5c3a1fd-20/s540x810/9d6ce72fcddf97841e7410a0652dd9d5f018b35d.pnj'
+      );
```

Comment:
```suggestion
      const imageUrl = 'https://64.media.tumblr.com/40e2174ab5e68b1eabbc3dfc78607cef/c1effc67d5c3a1fd-20/s540x810/9d6ce72fcddf97841e7410a0652dd9d5f018b35d.pnj'
      await sut({
        type: AvatarType.photo,
        image: imageUrl,
      });
      expect(screen.getByRole('img')).toHaveAttribute('src', imageUrl);
```
I think this way is easier to maintain.